### ISSUE TITLE
Minor bugs

### ISSUE THREAD

**Issue by golaz** [E3SM-Project/zppy#88]
A couple of minor bugs I encountered today:

1.  Resubmitting a previously failed global time series job may fail because directory already exists:
```
global_time_series_0051-0950
Traceback (most recent call last):
  File "/lcrc/soft/climate/e3sm-unified/base/envs/e3sm_unified_1.5.0rc5_chrysalis/bin/zppy", line 10, in <module>
    sys.exit(main())
  File "/lcrc/soft/climate/e3sm-unified/base/envs/e3sm_unified_1.5.0rc5_chrysalis/lib/python3.8/site-packages/zppy/__main__.py", line 88, in main
    global_time_series(config, scriptDir)
  File "/lcrc/soft/climate/e3sm-unified/base/envs/e3sm_unified_1.5.0rc5_chrysalis/lib/python3.8/site-packages/zppy/global_time_series.py", line 41, in global_time_series
    os.mkdir(c['global_time_series_dir'])
FileExistsError: [Errno 17] File exists: '/lcrc/group/e3sm/ac.golaz/E3SM_simulations/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis/post/scripts/global_time_series_0051-0950_dir'
```

2. Problem with dependency check if one of the dependency status file is not present. The job is not submitted (as it should), but a status file is nevertheless created, containing (WAITING -1).

https://github.com/E3SM-Project/zppy/blob/f12bd6fed92d82b223aaa4b8d3744f77f394f3db/zppy/utils.py#L131-L132

**Comment by forsyth2** [comment]
@golaz I included a fix for the first bug in #81. I can't replicate the second bug though. Do you have steps to follow?

**Comment by forsyth2** [comment]
@golaz I was able to replicate by running `climo`, `ts`, and `e3sm_diags` tasks, deleting a `climo` status file and all `e3sm_diags` bash/status files, setting `climo` to `active=False` in the configuration file, and then rerunning `zppy` on the configuration file.

Fixes for these bugs are included in #81.

**Comment by forsyth2** [comment]
Merging #81 resolves this.
